src/main/java/core/basesyntax/db/FileWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.db;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterImpl {
+    public void write(String report, String fileName) {
+        File file = new File(fileName);
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(file))) {
+            writer.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file" + fileName, e);

```

Comment:
The error message in the RuntimeException does not have a space between 'file' and the filename, this could lead to a misleading error message.

Reply:
ok, done